experimenter/experimenter/experiments/admin.py

```diff
@@ -109,9 +109,9 @@ def dehydrate_status_next(self, experiment):
         return experiment.status_next
 
     def dehydrate_qa_status(self, experiment):
-        """Return None instead of empty string for nullable enums"""
+        """Return NOT_SET instead of empty string for nullable enums"""
         if experiment.qa_status not in dict(NimbusConstants.QAStatus.choices):
-            return None
```

Comment:
@mikewilli I'm not sure if this is valid or not 🤔

Reply:
Looks fine to me. It's basically making sure the QA Status is valid so as long as what you return is valid that should work. My only concern is that it may be the string value that is needed, but you can add or update a test (and test locally) to figure that out. Probably worth exporting and importing a couple experiments locally too.

Reply:
Also this function may not even be necessary anymore because it is for nullable enums, and it looks like you changed the field to not be nullable anymore. If you want to remove it, you could test it out by exporting some experiments from Prod and importing them into your local instance, and then also export/import from/to local. I think it would break with the former but work with the latter (assuming migrations have been run locally but not yet on prod).

Reply:
awesome, I'll test that!

Reply:
Looks like it's not needed anymore! Both the import and export are working perfectly without the qa_status dehydration 💪